content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,189 @@
+==============
+Managing Lunch
```

Comment:
```suggestion
Lunch management
```
we're always trying to avoid using -ing words for the doc titles, so that's where this little suggestion is coming from 😄